proxy/README.md

```diff
@@ -0,0 +1,73 @@
+# IMAP Proxy
+
+A proxy that receives, `Debug`-prints, and forwards IMAP messages.
+
+It should™ forward all messages *unchanged*. Still, ...
```

Comment:
Technically this is not true by design. E.g. see `ServerFlowOptions::literal_accept_text` which will produce messages never sent by the real server.

Reply:
Grrr... does a footnote suffice? *Technically*, it's also not true on the syntax level as the IMAP syntax is not canonical, e.g., `a login` == `a LOGIN`. Important part is: It doesn't change the semantics.